controllers/csi/gc/binaries.go

```diff
@@ -0,0 +1,45 @@
+package csigc
+
+import (
+	"os"
+	"path/filepath"
+
+	dtcsi "github.com/Dynatrace/dynatrace-operator/controllers/csi"
+	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
+)
+
+func runBinaryGarbageCollection(logger logr.Logger, envID string, latestVersion string, opts dtcsi.CSIOptions) error {
+	gcPath := filepath.Join(opts.RootDir, dtcsi.DataPath, envID, dtcsi.GarbageCollectionPath)
+	gcDirs, err := os.ReadDir(gcPath)
+	if err != nil {
+		return errors.WithStack(err)
+	}
```

Comment:
There is a race condition upon startup when there isn't yet a version downloaded. Therefore, this shouldn't be an error in the first place. The condition can be logged as an info message.